src/components/Pool/PairsList/Pair/Pair.container.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useContext } from 'react'
+import { useIsMobileByMedia } from '../../../../hooks/useIsMobileByMedia'
+import { Flex } from 'rebass/styled-components'
+
+import { usePair24hVolumeUSD } from '../../../../hooks/usePairVolume24hUSD'
+import { formatCurrencyAmount } from '../../../../utils'
+import { unwrappedToken } from '../../../../utils/wrappedCurrency'
+
+import { ReactComponent as FarmingLogo } from '../../../../assets/svg/farming.svg'
+import DoubleCurrencyLogo from '../../../DoubleLogo'
+import { CurrencyLogo } from '../../../CurrencyLogo'
+import CarrotBadge from '../../../Badge/Carrot'
+import { useTranslation } from 'react-i18next'
+import { PairProps } from './Pair.types'
+import { EllipsizedText, FarmingBadge, GridCard, BadgeText } from './Pair.styles'
+import { ThemeContext } from 'styled-components'
+import { ResponsiveValueWithLabel } from './Pair.components'
+
+export function Pair({
+  token0,
+  token1,
+  usdLiquidity,
+  apy,
+  containsKpiToken,
+  pairOrStakeAddress,
+  hasFarming,
+  dayLiquidity,
+  isSingleSidedStakingCampaign,
+  ...rest
+}: PairProps) {
+  const { volume24hUSD, loading } = usePair24hVolumeUSD(pairOrStakeAddress, isSingleSidedStakingCampaign)
+  const { t } = useTranslation()
+  const theme = useContext(ThemeContext)
+  const isMobile = useIsMobileByMedia()
+
+  const correctLogo = () =>
+    isSingleSidedStakingCampaign ? (
+      <CurrencyLogo size={isMobile ? '32px' : '45px'} marginRight={14} currency={token0} />
+    ) : (
+      <DoubleCurrencyLogo
+        spaceBetween={0}
+        marginLeft={0}
+        marginRight={14}
+        top={0}
+        currency0={token0}
+        currency1={token1}
+        size={isMobile ? 32 : 45}
+      />
+    )
+
+  return (
+    <GridCard
+      {...rest}
+      data-testid="pair-card"
+      flexWrap="wrap"
+      justifyContent="space-between"
+      padding={isMobile ? '22px 10px' : '22px'}
+    >
+      <Flex flex={isMobile ? '' : '25%'} flexDirection="row" alignItems="center">
+        {correctLogo()}
+        <EllipsizedText color="white" lineHeight="20px" fontWeight="700" fontSize="16px" maxWidth="145px">
+          {unwrappedToken(token0)?.symbol}
+
+          {!isSingleSidedStakingCampaign && <br></br>}
```

Comment:
looks odd ;D
Also `br` tag is self-closing so you can reduce it to `{!isSingleSidedStakingCampaign && <br />}` but would be great to fix this by CSS

Reply:
It was already this way, but we should have cleaned it up. Thanks for the push.